app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1256,7 +1256,6 @@ class PatientRepositoryAndroidTest {
   fun verify_deleted_prescribed_drugs_are_not_included_when_fetching_recent_patients() {
     val facilityUuid = currentFacility.uuid
     val recentPatient1 = savePatientWithPrescribedDrug(facilityUuid = facilityUuid)
-    val recentPatient2 = savePatientWithPrescribedDrug(facilityUuid = facilityUuid, deletedAt = Instant.now())
```

Comment:
Why was this removed? This is needed because we want to ensure that deleted patients are not fetched in the recent patients query.